internal/api/v1beta1/user.go

```diff
@@ -148,7 +148,7 @@ func (h Handler) GetCurrentUser(ctx context.Context, request *shieldv1beta1.GetC
 		return nil, grpcPermissionDenied
 	}
 
-	if str.IsStringEmpty(email) {
+	if strings.TrimSpace(email) == "" {
```

Comment:
Do we do any other sanitisation steps? I don't know if this is user-provided value or not.

But usually, it's better to do it like this:
```golang
email = strings.TrimSpace(email)
if email == "" {
   return errors.New("invalid email")
}
```

Otherwise, I can send `"           foo@bar.com         "` and get stored.

Reply:
@spy16 This is user provided value. No, we mostly don't have any input checking in shield. Previously, for some domains you can even send an empty string as an id and stored in the DB.  I just added several points but maybe not covering all of them.
Agree on that, we better validate the input and return early or do sanitization. There should be another issue that focuses on this. For the email case in this PR, let me refactor as you suggested.

Reply:
yea. may be create an issue about validation so that we can pick it up separately. definitely better not to add this into scope of this PR 👍